src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slug: selectedPersonSlug } = useParams();
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.length
```

Comment:
It is better to use a conditional rendering instead of a ternary operator with an empty condition. You can change this line to: {people.length > 0 && ( ... )}

Reply:
I didn't get the point according empty condition :harold: 
If `people.length` is true I show the people, but if it is false (0) I show the corresponding message.

![image](https://user-images.githubusercontent.com/78492113/232853360-da95aba9-50e9-4f49-a061-badd82c4d5d3.png)

Reply:
I didn't get the point according empty condition :harold: 
If `people.length` is true I show the people, but if it is false (0) I show the corresponding message.

![image](https://user-images.githubusercontent.com/78492113/232853360-da95aba9-50e9-4f49-a061-badd82c4d5d3.png)

Reply:
My bad, everything is ok if you using ternary operator here

Reply:
But as for me better to refactor component a bit, move `<tr>` to separate component and use logical operators instead of ternary, 'cause such big block of code in ternary operator can confuse, like it was with me 😅